### PR TITLE
Bump snapshot and dev versions of libraries

### PR DESCRIPTION
**Pull request by reez**
### Description

Step 31 in [Issue](https://github.com/bitcoindevkit/bdk-ffi/issues/606) for beta 5 release.

#### All Submissions:

* [x] I've signed all my commits
* [x] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
* [ ] I ran `cargo fmt` and `cargo clippy` before committing

#### New Features:

* [ ] I've added tests for the new feature
* [ ] I've added docs for the new feature

#### Bugfixes:

* [ ] This pull request breaks the existing API
* [ ] I've added tests to reproduce the issue which are now passing
* [ ] I'm linking the issue being fixed by this PR

### CODE REVIEW

**Comment by reez** [comment]
@thunderbiscuit I can also update [this line](https://github.com/bitcoindevkit/bdk-ffi/blob/bfb2ae09ee9bc6676249011f1a825fecf337ef4c/.github/ISSUE_TEMPLATE/release.md?plain=1#L91) to say something different since I'm not sure we have those two templates anymore (maybe replace with `release`)?

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ git push upstream v0.6.0
 28.  - [ ] Aggregate all the changelog notices from the PRs and add them to the changelog file
 29.  - [ ] Bump the versions on master from `0.9.0-SNAPSHOT` to `0.10.0-SNAPSHOT`, `0.6.0.dev0` to `0.7.0.dev0`
 30.  - [ ] Apply changes to the minor_release and patch_release issue templates if they need any
-31.  - [ ] Open a PR on master with the changes in steps 29, 30, and 31. See [example PR here](https://github.com/bitcoindevkit/bdk-ffi/pull/317). Get a review and merge the PR.
+31.  - [ ] Open a PR on master with the changes in steps 28, 29, 30, and 31. See [example PR here](https://github.com/bitcoindevkit/bdk-ffi/pull/317). Get a review and merge the PR.
```
Comment (thunderbiscuit) [inline .github/ISSUE_TEMPLATE/release.md:92]: leaving 31 here is sort of self-referencing. I suggest to only keep 28, 29, 30.

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ git push upstream v0.6.0
 28.  - [ ] Aggregate all the changelog notices from the PRs and add them to the changelog file
 29.  - [ ] Bump the versions on master from `0.9.0-SNAPSHOT` to `0.10.0-SNAPSHOT`, `0.6.0.dev0` to `0.7.0.dev0`
 30.  - [ ] Apply changes to the minor_release and patch_release issue templates if they need any
```
Comment (thunderbiscuit) [inline .github/ISSUE_TEMPLATE/release.md:91]: Agree with your comment above. Let's fix this line.

**Review by thunderbiscuit — CHANGES_REQUESTED** [review]
Super small changes required on the release issue template but otherwise this looks great. I also think you might need a rebase on master.

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (reez) [reply .github/ISSUE_TEMPLATE/release.md]: removed 31, then just re-numbered the steps that followed-

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (reez) [reply .github/ISSUE_TEMPLATE/release.md]: fixed

**Comment by reez** [comment]
> Super small changes required on the release issue template but otherwise this looks great. I also think you might need a rebase on master.

Changes made & rebased-

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (thunderbiscuit) [reply .github/ISSUE_TEMPLATE/release.md]: Oh I see. Yep I re-read what I wrote and I see how it sounded now. Yeah that works too!

Comment (thunderbiscuit) [reply .github/ISSUE_TEMPLATE/release.md]: I think it's fairly clear those changes need to be merged. As long as it's clear to you as well that they need to be merged on master then it's cool.

**Review by thunderbiscuit — APPROVED** [review]
ACK ed2aadf578dc295a7614853371ae639fd1be4801.

File: .github/ISSUE_TEMPLATE/release.md (hunk 1 above)
Comment (reez) [reply .github/ISSUE_TEMPLATE/release.md]: Yeah I think its clear to me now but I'll keep it in mind, but yeah I read it one way at first and updated the PR and re-read it now and it makes sense in a different way. Will merge this PR, but will keep in mind if it doesnt make sense next time to add it back or re-word it

**Pull request merged**